src/controllers/invoices.js

```diff
@@ -98,7 +98,7 @@ export default function () {
           ${(shippingLine
             ? `
             ${i18n(i19zipCode)}: ${formatCEP(shippingLine.to.zip)}<br>
-            <address>${getLineAddress(shippingLine.to)}</address>`
+            <address>${getLineAddress(shippingLine.to)} ${shippingLine.to && shippingLine.to.near_to ? `<br>Referência: ${shippingLine.to.near_to}` : ''}</address>`
```

Comment:
```suggestion
            <address>${getLineAddress(shippingLine.to)} ${(shippingLine.to.near_to ? `<br>Referência: ${shippingLine.to.near_to}` : '')}</address>`
```

não tem como chegar nesse ponto aí se `shippingLine.to` undefined...